pkg/pool-hooks/contracts/StableSurgeHook.sol

```diff
@@ -285,6 +367,25 @@ contract StableSurgeHook is BaseHooks, VaultGuard, SingletonAuthentication {
             );
     }
 
+    function _isSurging(
+        SurgeFeeData memory surgeFeeData,
+        uint256[] memory currentBalances,
+        uint256 newTotalImbalance
+    ) internal pure returns (bool isSurging) {
+        // If we are balanced, or the balance has improved, do not surge: simply return the regular fee percentage.
+        if (newTotalImbalance == 0) {
+            return false;
+        }
+
+        uint256 oldTotalImbalance = StableSurgeMedianMath.calculateImbalance(currentBalances);
+
+        if (newTotalImbalance <= oldTotalImbalance || newTotalImbalance <= surgeFeeData.thresholdPercentage) {
+            return false;
+        }
+
+        return true;
```

Comment:
```suggestion
        return newTotalImbalance > oldTotalImbalance && newTotalImbalance > surgeFeeData.thresholdPercentage;
```

Reply:
yeah I left it as it was to minimize changes, but this way is probably better.